recipe/build.sh

```diff
@@ -109,10 +109,18 @@ if [[ ${cuda_compiler_version} != "None" ]]; then
         export TF_CUDA_PATHS="${BUILD_PREFIX}/targets/x86_64-linux,${PREFIX}/targets/x86_64-linux"
 	# XLA can only cope with a single cuda header include directory, merge both
 	rsync -a ${PREFIX}/targets/x86_64-linux/include/ ${BUILD_PREFIX}/targets/x86_64-linux/include/
+
+	# hmaarrfk -- 2023/12/30
+        # This logic should be safe to keep in even when the underlying issue is resolved
+        if [[ -x ${BUILD_PREFIX}/nvvm/bin/cicc ]]; then
+            cp ${BUILD_PREFIX}/nvvm/bin/cicc ${BUILD_PREFIX}/bin/cicc
+        fi
```

Comment:
you need to add the corresponding "rm" command right? otherwise you are modifying the user's BUILD_PREFIX.

I guess it doesn't appear in conda file, but I find it "good form"

Reply:
is there a better way to specify the location of CICC?

Reply:
I treat the `BUILD_PREFIX` as "trashed" as it will be destroyed after the build anyways. Especially as we are polluting it with a lot of bazel stuff, I don't think it is worth to care about cleanup (in contrast to $PREFIX!)

Reply:
Nothing binding, just curiosity:

Can we add a reference to the "underlying issue"? Is this about the CUDA 11.8 & CUDA 12 setup divergence? Do we really want to copy instead of symlink?

Reply:
copy vs symlink really doesn't matter. copy is more "resilient".

I added the reference by pushing a commit